.cirrus.yml

```diff
@@ -6,9 +6,7 @@ env:
     CIRRUS_SHELL: "/bin/bash"
     # No need to go crazy, but grab enough to cover most PRs
     CIRRUS_CLONE_DEPTH: 10
-    # Prefix git URL for every container tool's repo home
-    REPO_PREFIX: https://github.com/containers
-    IMAGE_SUFFIX: "c20240320t153921z-f39f38d13"
+    IMAGE_SUFFIX: "c20240411t124913z-f39f38d13"
```

Comment:
DANGER! `c20240411t124913z` is from https://github.com/containers/automation_images/pull/340 which is my `/var/tmp` change which has not been approved. Until (unless) that is approved and distributed to the other tools, that ID should be considered toxic.

Reply:
Noted.  I simply grabbed it because it was recent.  I'll fine another one that was merged.